test/e2e/framework.go

```diff
@@ -1577,7 +1578,39 @@ func (data *TestData) PodWaitFor(timeout time.Duration, name, namespace string,
 // until the provided timeout expires).
 func (data *TestData) podWaitForRunning(timeout time.Duration, name, namespace string) error {
 	_, err := data.PodWaitFor(timeout, name, namespace, func(pod *corev1.Pod) (bool, error) {
-		return pod.Status.Phase == corev1.PodRunning, nil
+		if pod.Status.Phase != corev1.PodRunning {
+			return false, nil
+		} else {
+			node, err := data.clientset.CoreV1().Nodes().Get(context.TODO(), pod.Spec.NodeName, metav1.GetOptions{})
+			if err != nil {
+				return false, fmt.Errorf("error when getting Node for Pod '%s': %v", name, err)
+			}
+
+			osType := node.Status.NodeInfo.OperatingSystem
+			runtime := node.Status.NodeInfo.ContainerRuntimeVersion
+			if osType == "windows" && strings.Contains(runtime, "containerd") {
+				var stdout string
+				antctlCmd := fmt.Sprintf("C:/k/antrea/bin/antctl.exe get podinterface %s -n %s -o json", name, namespace)
```

Comment:
add a comment to explain that `C:/k/antrea/bin/antctl.exe` is guaranteed to be installed when installing Antrea on Windows Nodes using the Antrea YAML manifests

Reply:
Updated.